core/partitioning/partitioning.cpp

```diff
@@ -111,7 +115,7 @@ void find_all_fallback_nodes(std::unordered_map<torch::jit::Node*, int>& fallbac
     // for every node that produces this fallback node's NonTensor input, they should fallback too
     for (auto input : cur_node->inputs()) {
       if (!isTensor(input) && input->node()->kind() != torch::jit::prim::Constant &&
-          fallback_nodes.insert({input->node(), 4}).second) {
+          global_fallback_nodes.insert({input->node(), 4}).second) {
```

Comment:
nit: Consider an enum rather than using raw values for fallback reason.

Reply:
good idea! you mean we should use enum rather than raw values like 4 here right? thanks!